activesupport/lib/active_support/json/decoding.rb

```diff
@@ -28,6 +28,7 @@ def decode(json)
           data
         end
       end
+      alias_method :load, :decode
```

Comment:
Do we still need this?

Reply:
We discussed that when implementing and weren't sure. It's nice to have since it allows users to specify a JSON serializer without having to make their own wrapper, but also now that's kinda a different feature.

If you think it should be removed, then we can 🔥 it

Reply:
>  it allows users to specify a JSON serializer without having to make their own wrapper

I agree with this, but I also agree with
> also now that's kinda a different feature

So I think it's safer to take it out. 🤔

Reply:
I'm not terribly familiar with the etiquette on Rails PRs, if we think those have value we should keep them in. 

Easy to explain why they are there, also easy to 🔥 if they say no.